ansible/roles/infra-osp-resources-destroy/tasks/delete_osp_resources.yml

```diff
@@ -29,6 +29,14 @@
           command: |
             openstack project purge --keep-project --project {{ osp_project_name }}
           when: project_purge_out is succeeded
+
+        - name: Delete any remaining volumes in project
+          command: |
+            openstack volume list --project {{ osp_project_name }} -f value -c ID | xargs openstack volume delete
```

Comment:
If you're using pipe, then  you need `shell` features.
`command` will fail here.